src/Files.App (Package)/Files.Package.wapproj

```diff
@@ -104,7 +104,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.3.230724000" IncludeAssets="build" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230518007-experimental1" IncludeAssets="build" />
```

Comment:
Is experimental needed here?

Reply:
We'll have to wait for 1.4 to reach stable before merging, in the meantime, I'm wondering if this is causing the pipeline to crash.

Reply:
No guarantee that 1.4 stable will have that API.. it's been experimental since 1.1. Will check again if stable can be used.

Reply:
WinAppSdk 1.4 Preview 2 is now available.

Reply:
But sadly does not include ContentExternalOutputLink.